server/utils/matchUtils.ts

```diff
@@ -145,6 +150,48 @@ export const confirmationSummaryCardRows = (
   ]
 }
 
+export const spaceBookingSummaryCardRows = (
+  premisesName: string,
+  apType: ApType,
+  dates: PlacementDates,
+  gender: Gender,
+  essentialCharacteristics: Array<PlacementCriteria>,
+  desirableCharacteristics: Array<PlacementCriteria>,
+): Array<SummaryListItem> => {
+  return [
+    summaryCardTextRow('Approved Premises', premisesName),
+    summaryCardTextRow('AP Type', apTypeLabels[apType]),
+    arrivalDateRow(dates.startDate),
+    departureDateRow(dates.endDate),
+    placementLengthRow(dates.placementLength),
+    summaryCardTextRow('Gender', sentenceCase(gender)),
+    summaryCardHtmlRow('Essential characteristics', requirementsHtmlString(essentialCharacteristics)),
+    summaryCardHtmlRow('Desirable characteristics', requirementsHtmlString(desirableCharacteristics)),
+  ]
+}
+
+export const filterOutAPTypes = (requirements: Array<PlacementCriteria>) => {
```

Comment:
I think this should replace the `filterPlacementCriteriaToSpaceCharacteristics` currently in main, it does the same thing but this name is better